api/libproject/devstart.rb

```diff
@@ -587,3 +629,11 @@ def run
   :description => "Set user authorities (permissions). See set-authority --help.",
   :fn => lambda { |*args| SetAuthority.new("set-authority", args, true).run }
 })
+
+Common.register_command({
+  :invocation => "deploy-api",
+  :description => "Deploys the API server to the specified cloud project.",
+  :fn => lambda { |*args| DeployApi.new("deploy-api", args, false).run }
```

Comment:
The default for the third constructor arg is `false`, so you can omit it.

Reply:
Done.